include/deal.II/numerics/vector_tools.h

```diff
@@ -771,6 +771,26 @@ namespace VectorTools
    *
    * In 1d, the default value of the boundary quadrature formula is an invalid
    * object since integration on the boundary doesn't happen in 1d.
+   *
+   * @note This function is not implemented for MPI parallel computations,
+   * see step-32 for a way to do a projection in parallel.
+   *
+   * @param mapping The mapping object to use.
+   * @param dof The DoFHandler the describes the finite element space to
+   * project into and that corresponds to @p vec.
+   * @param constraints Constraints to be used when assembling the mass
+   * matrix, typically needed when you have hanging nodes.
+   * @param quadrature The quadrature formula to be used for assembling the
+   * mass matrix.
+   * @param function The function to project into the finite element space.
+   * @param[out] vec The output vector where the projected function will be
```

Comment:
A combination of laziness and the thought that this just generates additional noise ("in" is a good default assumption). I can add those if you prefer.